src/main/resources/plugin.yml

```diff
@@ -270,6 +273,7 @@ permissions:
       simpleclans.member.bb: true
       simpleclans.member.bb-add: true
       simpleclans.member.bb-toggle: true
+      simpleclans.member.invite-toggle: true

```

Comment:
Don't you already have a `simpleclans.anyone.invite-toggle`?

Reply:
i forget this